charts/portworx/values.yaml

```diff
@@ -115,3 +115,14 @@ clusterToken:
 
 deleteStrategy:                           # Optional: Delete strategy for the portworx cluster
   type:                                   # Valid values: Uninstall, UninstallAndWipe
+
+updateStrategy:   
+  type:                                   # portworx supports the following update strategies- RollingUpdate and OnDelete
```

Comment:
should we add RollingUpdate as default?

Reply:
We are only adding the updateStrategy in storageCluster if the user has specified valid values. The operator can handle the default values later if we not specifies any value in type.